src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -203,6 +214,9 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
     contextPageOwnerSlug,
   ])
 
+  if (verificationStatus === "VERIFIED_NO_CHANGE")
+    return <div data-testid="emptyAddressVerification"></div>
+
   if (addressOptions.length === 0) return null
```

Comment:
This part differs a bit from [how #12636 handled it](https://github.com/artsy/force/pull/12636/files#diff-a65d5b9bf3f3ba0af862f0c53c8ea2e0be243adf95b86daacdb768a05e3f538aR127-R129). I'm not sure which is right.